waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -861,8 +876,32 @@ proc mountRlnRelay*(node: WakuNode, conf: WakuNodeConf) {.raises: [Defect, Value
       let memKeyPair = keyPair.toMembershipKeyPairs()[0]
       # mount the rln relay protocol in the on-chain/dynamic mode
       waitFor node.mountRlnRelayDynamic(memContractAddr = ethMemContractAddress, ethClientAddr = ethClientAddr, memKeyPair = some(memKeyPair), memIndex = some(rlnRelayIndex), ethAccAddr = ethAccountAddr, ethAccountPrivKey = ethAccountPrivKey, pubsubTopic = conf.rlnRelayPubsubTopic, contentTopic = conf.rlnRelayContentTopic)
+    elif fileExists("keyPair.txt") and fileExists("rlnIndex.txt"):
```

Comment:
I may not understand the exact context, but if the idea is simply for this "write to cleartext file" to be a temporary stopgap until we have an integrated keystore, the logic should be abstracted to separate functions with no magic references to the .txt files in the main logic.

In other words, instead of having an `elif fileExists(...` here, it should be something like `elif usePersistentKeys()` where `usePersistentKeys()` are implemented elsewhere. And instead of having the logic to read the keys from the files here it should be abstracted to a `readKeys()` proc, etc.

Reply:
Suggestion incorporated. refer to 6236dd6fa5501e5ae1e7b115138bcd1176534d73